examples/stereo_depth_video.py

```diff
@@ -1,116 +1,273 @@
 #!/usr/bin/env python3
 
 import cv2
-import depthai as dai
 import numpy as np
+import depthai as dai
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-res",
+    "--resolution",
+    type=str,
+    default="720",
+    help="Sets the resolution on mono cameras. Options: 800 | 720 | 400",
+)
+parser.add_argument(
+    "-md",
+    "--mesh_dir",
+    type=str,
+    default=None,
+    help="Output directory for mesh files. If not specified mesh files won't be saved",
+)
+parser.add_argument(
+    "-lm",
+    "--load_mesh",
+    default=False,
+    action="store_true",
+    help="Read camera intrinsics, generate mesh files and load them into the stereo node.",
+)
+parser.add_argument(
+    "-rect",
+    "--out_rectified",
+    default=False,
+    action="store_true",
+    help="Generate and display rectified streams",
+)
+parser.add_argument(
+    "-lr",
+    "--lrcheck",
+    default=False,
+    action="store_true",
+    help="Better handling for occlusions",
+)
+parser.add_argument(
+    "-e",
+    "--extended",
+    default=False,
+    action="store_true",
+    help="Closer-in minimum depth, disparity range is doubled",
+)
+parser.add_argument(
+    "-s",
+    "--subpixel",
+    default=False,
+    action="store_true",
+    help="Better accuracy for longer distance, fractional disparity 32-levels",
+)
+parser.add_argument(
+    "-m",
+    "--median",
+    type=str,
+    default="7x7",
+    help="Choose the size of median filtering. Options: OFF | 3x3 | 5x5 | 7x7 (default)",
+)
+parser.add_argument(
+    "-d",
+    "--depth",
+    default=False,
+    action="store_true",
+    help="Display depth frames",
+)
+args = parser.parse_args()
+
+resolutionMap = {"800": (1280, 800), "720": (1280, 720), "400": (640, 400)}
+if args.resolution not in resolutionMap:
+    exit("Unsupported resolution!")
+
+resolution = resolutionMap[args.resolution]
+mesh_directory = args.mesh_dir  # Output dir for mesh files
```

Comment:
use consistent naming convention, camelCase instead snake_case throughout the code